### PR TITLE
Add compat data for SVG <line>

### PR DESCRIPTION
**Pull request by JeremiePat**
https://developer.mozilla.org/en-US/docs/Web/SVG/Element/line

### CODE REVIEW

File: svg/elements/line.json [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+                "version_added": "1"
+              },
+              "chrome_android": {
+                "version_added": true
+              },
+              "edge": {
+                "version_added": true
+              },
+              "edge_mobile": {
+                "version_added": null
```
Comment (SebastianZ) [inline svg/elements/line.json]: Should be `true`, as Edge mobile generally supports `<line>` and it's a default attribute you have to set for lines.

File: svg/elements/line.json [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+                "version_added": "4"
+              },
+              "ie": {
+                "version_added": "9"
+              },
+              "opera": {
+                "version_added": "8"
+              },
+              "opera_android": {
+                "version_added": null
```
Comment (SebastianZ) [inline svg/elements/line.json]: Should be `true`, as Opera for Android generally supports `<line>` and it's a default attribute you have to set for lines.

File: svg/elements/line.json [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+                "version_added": "1"
+              },
+              "chrome_android": {
+                "version_added": true
+              },
+              "edge": {
+                "version_added": true
+              },
+              "edge_mobile": {
+                "version_added": null
```
Comment (SebastianZ) [inline svg/elements/line.json]: Same here.

File: svg/elements/line.json [hunk 4]
```diff
@@ -0,0 +1,242 @@
...
+                "version_added": "4"
+              },
+              "ie": {
+                "version_added": "9"
+              },
+              "opera": {
+                "version_added": "8"
+              },
+              "opera_android": {
+                "version_added": null
```
Comment (SebastianZ) [inline svg/elements/line.json]: Same here.

File: svg/elements/line.json [hunk 5]
```diff
@@ -0,0 +1,242 @@
...
+                "version_added": "1"
+              },
+              "chrome_android": {
+                "version_added": true
+              },
+              "edge": {
+                "version_added": true
+              },
+              "edge_mobile": {
+                "version_added": null
```
Comment (SebastianZ) [inline svg/elements/line.json]: Same here.

File: svg/elements/line.json [hunk 6]
```diff
@@ -0,0 +1,242 @@
...
+                "version_added": "4"
+              },
+              "ie": {
+                "version_added": "9"
+              },
+              "opera": {
+                "version_added": "8"
+              },
+              "opera_android": {
+                "version_added": null
```
Comment (SebastianZ) [inline svg/elements/line.json]: Same here.

File: svg/elements/line.json [hunk 7]
```diff
@@ -0,0 +1,242 @@
...
+                "version_added": "1"
+              },
+              "chrome_android": {
+                "version_added": true
+              },
+              "edge": {
+                "version_added": true
+              },
+              "edge_mobile": {
+                "version_added": null
```
Comment (SebastianZ) [inline svg/elements/line.json]: Same here.

File: svg/elements/line.json [hunk 8]
```diff
@@ -0,0 +1,242 @@
...
+                "version_added": "4"
+              },
+              "ie": {
+                "version_added": "9"
+              },
+              "opera": {
+                "version_added": "8"
+              },
+              "opera_android": {
+                "version_added": null
```
Comment (SebastianZ) [inline svg/elements/line.json]: Same here.

**Comment by JeremiePat** [comment]
All fixed :)

**Pull request merged**